src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {
+      person.age
+        && (
+          <p className="Person__age">
+            {`I am ${person.age}`}
+          </p>
+        )
+    }
+
+    <p className="Person__partner">
+      {
+        person.isMarried
```

Comment:
Code readability can be improved by not using nested ternary operators. Consider refactoring this logic into a separate function.